apps/ng-doc/project.json

```diff
@@ -45,7 +45,8 @@
 					"posthtml-render",
 					"js-beautify",
 					"chalk"
-				]
+				],
+				"ngDoc": { "config": "apps/ng-doc/ng-doc.config.ts" }
```

Comment:
I wouldn't want to set the path to the main configuration here; the configuration will be found by default using the search mechanism. This can also be considered an additional check that the default search mechanism the library uses for each project is working.

Reply:
Yeah sure. Let's remove it then. I added this to test if the fix works.

Reply:
Yeah sure, let's remove it then. I added this to test if the fix works,